lib/actions/ui.js

```diff
@@ -202,37 +204,69 @@ export function moveTo(i) {
   };
 }
 
+function getDefaultJsEditor() {
+  const regKey = '\\JSFile\\Shell\\Edit\\Command';
+  new Registry({
+    hive: 'HKCR',
+    key: regKey
+  }).get(Registry.DEFAULT_VALUE, (err, entry) => {
+    if (err) {
+      // In case of error, return default editor
+      return 'notepad.exe';
+    }
+    // Default is "C:\Windows\System32\Notepad.exe %1"
+    entry = entry.substring(0, entry.indexOf(' %1'));
+    entry = entry.substring(entry.lastIndexOf('\\'), entry.length);
+    return entry;
+  });
+}
+
 export function showPreferences() {
   const isWin = process.platform === 'win32';
   // eslint-disable-next-line no-template-curly-in-string
-  const command = isWin ? ' start notepad "%userprofile%\\.hyper.js"' : ' bash -c \'exec env ${EDITOR:=nano} ~/.hyper.js\'';
-  const message = [];
-  message.push(isWin ?
-    ' echo Attempting to open ^%userprofile^%\\.hyper.js with notepad' :
-    ' echo Attempting to open ~/.hyper.js with your \\$EDITOR');
-  message.push(' echo If it fails, open it manually with your favorite editor!');
+  let command = ' bash -c \'exec env ${EDITOR:=nano} ~/.hyper.js\'';
```

Comment:
Okay, I'm convinced. I'll update my pull request. 

I'm building a Windows VM to test locally too so I can test - my bare metal machines all have multiple versions of Visual Studio and Python, which is not playing well with the build.